code/css/style.css

```diff
@@ -0,0 +1,198 @@
+/* style.css */
+
+/* setup */
+* {
+  box-sizing: border-box;
+}
+
+/* clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+  clear:both;
+}
+
+.alignleft {
+  float: left;
+}
+
+.alightright {
+  float: right;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+/* lyout */
+.container {
+  width: 960px;
+  margin: 0 auto;
+}
+
+body {
+  color: ;
+  background: #A5B8A7;
+  font-family: Amatic SC;
+  font-size:;
+  font-weight:;
+  line-height:;
+}
+
+a {
+  text-decoration: none;
+}
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+header {
+  padding: 25px 0;
+  overflow:hidden;
+  background:#ED5A36;
+}
+
+header nav {
+  float: right;
+  margin-top: 15px;
+}
+
+header nav ul li {
+  display: inline-block;
+  margin: 0 0 0 40px;
```

Comment:
If the rest of the values are 0, you could also just try using `margin-bottom: 40px`